models/comments.models.js

```diff
@@ -12,9 +12,30 @@ function selectCommentsByArticleId(article_id) {
         WHERE article_id = $1
         ORDER BY created_at DESC;
     `, [article_id])
-    .then(result => {
+    .then((result) => {
         return result.rows; 
     });
 }
 
-module.exports = { selectCommentsByArticleId };
+//task 7
+function insertCommentByArticleId(article_id,username,body){
+    if (isNaN(article_id)) {
```

Comment:
you dont need to check that the article_id is invalid here, sql will do this for you